base/essentials.jl

```diff
@@ -158,6 +158,11 @@ macro boundscheck(blk)
       $(Expr(:boundscheck,:pop)))
 end
 
+"""
+    @inbounds(blk)
+
+Tells the compiler to skip bound checks within the given block.
```

Comment:
Ah right that's devdocs, so maybe more detailed than most users need. Examples are always good, and could have that cross reference but say something like "for advanced usage see ..."